src/App.vue

```diff
@@ -272,17 +275,20 @@
     },
   }
 </script>
-<style>
 
+<style>
 .mapboxgl-ctrl-top-right {
-    top: 0;
-    right: 0;
+  top: 0;
+  right: 0;
 }
 
 @media only screen and (max-width:1199px) {
   .mapboxgl-ctrl-top-right {
     top: 0;
-    right: calc(100vw - 560px);
+    right: calc(100vw - 600px);
   }
+  .mapboxgl-ctrl-top-right .mapboxgl-ctrl {
+    float: left;
   }
+}
```

Comment:
Both the float and the increase on the right are needed to have it look like this
![image](https://github.com/openearth/rws-viewer/assets/15196342/7ee4d1d7-5328-4c51-9eeb-8d2e8811dd3f)